static/app/components/eventOrGroupHeader.tsx

```diff
@@ -182,7 +181,7 @@ const TitleWithoutLink = styled('span')`
   display: inline-flex;
 `;
 
-export default withOrganization(EventOrGroupHeader);
+export default EventOrGroupHeader;
```

Comment:
Looks like there are a couple places which weren't providing organization as a prop that are angry

Reply:
oops fixing now, didn't mean to include this in the PR